homeassistant/components/media_player/songpal.py

```diff
@@ -36,6 +40,7 @@
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME): cv.string,
     vol.Required(CONF_ENDPOINT): cv.string,
+    vol.Optional(CONF_POLL, default=False): cv.boolean,
```

Comment:
Does it need to be configurable?

Reply:
Yeah, some implementations do not offer the websockets interface, so that could probably done programatically (if it's worth the effort; those that are definitely incompatible are the bravia televisions, which are supported by another platform). I'll remove the setting, but leave the variable in place. If someone encounters problems they can adapt the source code and/or the autodetection can be fixed to avoid websockets afterwards. Thanks for your reviews and help!

Reply:
Done, updated also the description & closed the documentation PR accordingly.